tools/libfrencutils/mosaic_util.c

```diff
@@ -721,14 +736,14 @@ void mult(long double m[], long double v[], long double out_v[]) {
 /* returns 1 if matrix is inverted, 0 otherwise */
 int invert_matrix_3x3(long double m[], long double m_inv[]) {
 
-  
+
   const long double det =  m[0] * (m[4]*m[8] - m[5]*m[7])
                      -m[1] * (m[3]*m[8] - m[5]*m[6])
                      +m[2] * (m[3]*m[7] - m[4]*m[6]);
 #ifdef test_invert_matrix_3x3
   printf("det = %Lf\n", det);
-#endif  
-  if (fabs(det) < EPSLN15 ) return 0;
+#endif
+  if (fabsl(det) < EPSLN15 ) return 0;
```

Comment:
I think this needs to be reverted to fabs to work with the macro inserted into mosaic_util.h

Reply:
`det` is defined a few lines above as `const long double det`.  Since `det` is always `long double` this is the correct abs function.